### PR TITLE
Refresh media resources

### LINKED ISSUE #1896: Change to UGRC logos on media page
_Please be comfortable to submit an issue if you have a question, want to ask to post new content or if the website isn't working properly for you._

## Issue description
We renamed out agency and have new logos and need to replace our old logos on the media page.

#### What page is the problem visible
https://gis.utah.gov/about/media/

#### Expected result
I expected: 
To have downloadable UGRC "wavy gravy" logos

#### Actual result
Instead:
We are providing downloadable "tiara" AGRC logos

**Question:** which logos would we like to make available from 
https://drive.google.com/drive/folders/1JfTXMAsmH8vPaKUhtqBtT9MQPoufKVns 
and/or
https://drive.google.com/drive/folders/1bX0CR11C5ckH1AhePynHPZxhyvNInZZc

### PR DESCRIPTION
**Pull request by steveoh**
This helps move us along with our rebranding.

Pages to look at

- [media](https://deploy-preview-2048--gis-utah-gov.netlify.app/about/media)
- [data](https://deploy-preview-2048--gis-utah-gov.netlify.app/data)
- [sgid](https://deploy-preview-2048--gis-utah-gov.netlify.app/sgid)
- [turn](https://deploy-preview-2048--gis-utah-gov.netlify.app/gps/turn)

closes #1896

### CODE REVIEW

File: about/media/index.md
Comment (gregbunce) [inline about/media/index.md:69]: i like " let the data take you places" over " let data take you places".  but, I'm not stuck on it.  whatever you think.  that's my two cents.  i like the overall phrase.  nice work.

**Review by gregbunce — APPROVED** [review]
looks good! i checked a few out on mobile.  i didn't see any issues. 

the blogpost looks good, the data page looks good, the discover page looks good.  overall, it all like good.  

i made a minor suggestion of adding the word "the" to the API slogan, but that's a minor suggestion.  other than that, i like it!  i like the logos/branding on the pages. nice work.

**Comment by gregbunce** [comment]
i  guess my only other (and final) comment is that the data page and the sgid page have two different logos (one with the ugrc colors and one all in black).  i prefer both to be all in black but, I'm not willing to die on that hill.

**Comment by stdavis** [comment]
These look sweet! I love how they all share the same color scheme. That ties them together nicely.

I agree with Greg that the SGID logos should be the same on both pages, but I actually think that the colored version would be best.

**Comment by steveoh** [comment]
this needs an approval before merging.

File: about/media/index.md
Comment (steveoh) [reply about/media/index.md:69]: I don't feel like changing it all for a the. I'm going to leave it for now.

**Review by gregbunce — APPROVED** [review]

**Pull request merged**
